recipe/meta.yaml

```diff
@@ -6,28 +6,32 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/manhole-{{ version }}.tar.gz
   sha256: bada20a25b547b395d472e2e08928f0437df26bbdbda4797c55863198e29a21f
 
 build:
-  number: 3
   entry_points:
     - manhole-cli = manhole.cli:main
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [win]
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 4
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
   run:
-    - python
+    - python >=3.6
+    - __unix
```

Comment:
I am not familiar with the `__unix` package. What does it do and why do we need it here?

Reply:
It restricts the package to run only on Unix machines. The way I understand upstream, this package only works there so that makes sure that it cannot be installed on windows machines. 
When installing the package, the conda/mamba solver will inform the user that it is not available for this platform.

Reply:
Thanks for the explanation, @BastianZim. I can't find any help related to that at https://conda-forge.org/docs/search.html?q=__unix. Must be a hidden feature 🙂.

Maybe the @conda-forge/core folks could point us to the relevant docs regarding `__unix`? I wonder if there are other types of `__<...>` things that could be useful for other feedstocks.

Reply:
It's in the docs I linked above :) 

https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html

Reply:
My bad 🤦. Sorry for the noise. Looks good to me!

Reply:
No worries. Can you also merge the PR please?